Source/AppScaffolding/AppScaffolding.csproj

```diff
@@ -18,6 +18,6 @@
     <DebugType>embedded</DebugType>
   </PropertyGroup>
   <Target Name="PreBuild" BeforeTargets="PreBuildEvent">
-    <Exec Command="if $(Configuration) == Debug (&#xD;&#xA;dotnet bump-version revision AppScaffolding.csproj&#xD;&#xA;)" />
+    <Exec Command="if $(Configuration) == Debug (&#xD;&#xA;dotnet tool restore&#xD;&#xA;dotnet bump-version revision AppScaffolding.csproj&#xD;&#xA;)" />
```

Comment:
My PS script does the build and compresses the built files into a zip with your naming format using the version number from AppScaffolding.csproj.

I have no idea how you do the rest of your work (editing release notes, positing with the github api, etc), but I hope you can just shoehorn this build/zip into your workflow.  Let me know if/when you want some help.

Reply:
As promised, I'm finally getting around to looking at the new publish stuff.

Automatically, bumping version was once useful but really hasn't been for quite a while -- so it's fine that you removed MSBump. I also removed the DotnetBumpVersion `PreBuild` thing. And I also removed some other stuff by accident but just put it back -- oops.

I've now played around manually with `dotnet publish`. I'll likely not use the powershell script per se but I will use those 3 publish commands in my publishing tool which will take advantage of your new publish profiles. The smaller size is slick. I haven't adjusted that tool yet but I will. Currently it uses devenv to build at the solution level instead of the project level. `dotnet publish` needs project-level though so I'll need to modify my tool to do it that way.

Feel free to keep that powershell script in there if it's useful to you. It is a neat script.

Reply:
Cool. I'll be glad to see the smaller file sizes.  I'm not precious about the script, so just go ahead and delete it.

Reply:
> Cool. I'll be glad to see the smaller file sizes. I'm not precious about the script, so just go ahead and delete it.

Good to know. I'll keep it around until I have a suitable replacement.